### DISCUSSION TITLE
Multiple Umbracos

### DISCUSSION THREAD

**Discussion by pcg-abrown** [nathanwoulfe/Plumber-2#125] [Q&A]
The license allows all sub-domains, and this I understand - but do all sub-domains need to be on the same umbraco installation - or can you install plumber on multiple umbracos?

**Comment by nathanwoulfe** [comment]
That's a good question, and not something I'm 100% sure about...

The intent with sub-domains is to allow a single license to cover dev.mysite.com, test.mysite.com and www.mysite.com. 

The licensing logic checks the domain serving the Umbraco instance, so in theory should be fine if the sub domains are different Umbracos so long as the domain is the same across all.

While that's within the terms of the license, it's not intended to allow, for example, an agency to run 100 client BackOffices from sub-domains of myagency.com, on a single license.

Not much I can do about that given it's domain licensing, but should probably update the EULA to clarify acceptable usage. Example above is essentially on-selling, which isn't acceptable.
